README.md

```diff
@@ -1 +1,40 @@
-# hello-microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello Microverse
+
+> project on config linters
+
+
+## Built With
+
+- HTML - 5
+- CSS - 3
+
+
+## Author
+
+👤 Amit Kumar
+
+- GitHub: [@AmitKlohan](https://github.com/AmitKlohan)
+
+
+
```

Comment:
Please try to add more personal information like your LinkedIn profile, Twitter username, and also your location.